WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -45,6 +45,7 @@
 public class PeopleInviteFragment extends Fragment implements
         RoleSelectDialogFragment.OnRoleSelectListener,
         PeopleManagementActivity.InvitationSender {
+    private static final String URL_USER_ROLES_DOCUMENTATION = "https://en.support.wordpress.com/user-roles/";
```

Comment:
I doubt this will be reused elsewhere, so I think it's fine to keep it private here (same for `UNKNOWN_SEARCH_TERMS_HELP_PAGE`)